invenio_rdm_records/collections/models.py

```diff
@@ -156,9 +156,30 @@ def get_by_slug(cls, slug, tree_id):
         return cls.query.filter(cls.slug == slug, cls.tree_id == tree_id).one_or_none()
 
     @classmethod
-    def read_many(cls, ids):
+    def read_many(cls, ids_):
         """Get many collections by ID."""
-        return cls.query.filter(cls.id.in_(ids)).order_by(cls.path, cls.order)
+        return cls.query.filter(cls.id.in_(ids_)).order_by(cls.path, cls.order)
+
+    @classmethod
+    def read_all(cls):
+        """Get all collections."""
+        return cls.query.order_by(cls.path, cls.order)
+
+    def update(
+        self, /, slug=None, title=None, search_query=None, order=None, num_records=None
```

Comment:
What is the `/` param? Did you mean `_`?

Reply:
I modified the method now based on your suggestions above. 

Anyway, I wanted to use **keyword-only** [parameters](https://docs.python.org/3/whatsnew/3.8.html#positional-only-parameters) (`*`) and not `positional-only`. The [idea](https://github.com/inveniosoftware/invenio-rdm-records/pull/1853#discussion_r1808909774) was that if someone uses `collection.update` they need to be explicit of what they want to update, to avoid unexpected errors coming from e.g. swapping the order unintentionally or drilling objects.

Since I now refactored the `update` code to have a `set` of allowed parameters, it accomplishes the same and this was removed.